reactioncommerce/reaction-devtools
# [WIP] Use reaction-data-loader to load data

`reaction-data-loader` is used to load data from now on.
The UI looks like this ->
<img width="258" alt="screen shot 2018-07-02 at 3 35 00 pm" src="https://user-images.githubusercontent.com/7762605/42159373-306640d2-7e11-11e8-9b95-58081e2aa090.png">

## To test
1. Clone the repo into `imports/plugin/custom/`
2. Run `meteor npm install`
3. Run `reaction reset -n`
4. Run `reaction`
5. Navigate to `localhost:3000` and login as admin.
6. Open devtools and select "Small Dataset", and then "Load data"
7. See the progress in the console.
8. Refresh to see all the products.
9. Repeat steps 6-8 with different datasets(loading the large dataset will take a lot of time.)
10. Try loading the swagshop data with "Load sample dataset".
